pkg/lib/timeformat.ts

```diff
@@ -41,11 +33,32 @@ export const dateTimeNoYear = (t: Time): string => formatter({ month: 'short', d
 // Wednesday, June 30, 2021
 export const weekdayDate = (t: Time): string => formatter({ dateStyle: "full" }).format(t);
 
-// about 1 hour [ago]
-export const distanceToNow = (t: Time, addSuffix?: boolean): string => formatDistanceToNow(t, {
-    locale: dateFnsLocale(),
-    addSuffix: addSuffix ?? false
-});
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/format
+const unitNames: Intl.RelativeTimeFormatUnit[] = ["second", "minute", "hour", "day", "week", "month", "year"];
```

Comment:
I think I'd prefer to see this array done more like

```ts
const units = [
  [ 1, "second" ],
  [ 60, "minute" ],
  [ 3600, "hour" ],
  [ 86400, "day" ],
  ...
];
```

or even

```ts
const units = [
  [ 1, "second" ],
  [ 60, "minute" ],
  [ 60, "hour" ],
  [ 24, "day" ],
  ...
];
```

(ie: the factors get multiplied as you go, or equivalently, the number of seconds get divided as you go).

Not a veto.  Just an idea.

Reply:
Right, that reads nicer. I didn't put too much effort into the copy pasta, but I'm happy to rework it.

Reply:
I beautified that a little bit.